src/main/java/org/opentripplanner/updater/spi/HttpHeaders.java

```diff
@@ -0,0 +1,87 @@
+package org.opentripplanner.updater.spi;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Objects;
+import org.opentripplanner.standalone.config.framework.project.EnvironmentVariableReplacer;
+
+/**
+ * Encapsulate HTTP header parameters for use in updaters. The builder
+ * will use the {@link EnvironmentVariableReplacer} to substitute TOKENS in the
+ * values passed in. There is also a convenience method for adding
+ * {@link Builder#acceptApplicationXML()}.
+ */
+public class HttpHeaders {
+
+  private final Map<String, String> headers;
+
+  private HttpHeaders(Builder builder) {
+    this.headers = Map.copyOf(builder.headers);
+  }
+
+  public static Builder of(String source) {
```

Comment:
What benefit is there for this `source`?

Reply:
Thank you for spotting this. It was used for error logging when I did the Env.resolving inside the HttpHeader, but that code is removed and I forgot to remove this - it is no longer used. I will remove it.